galasa-parent/dev.galasa.framework.api.cps/src/main/java/dev/galasa/framework/api/cps/internal/routes/CPSRoute.java

```diff
@@ -200,10 +128,20 @@ protected String buildResponseBody(String namespace, Map.Entry<String, String> e
         if (entry != null){
             JsonObject cpsProp = new JsonObject();
             cpsProp.addProperty("name", entry.getKey());
-            cpsProp.addProperty("value", getProtectedValue(entry.getValue(),namespace));
+            cpsProp.addProperty("value", propertyUtility.getProtectedValue(entry.getValue(),namespace));
             propertyArray.add(cpsProp);
         }
         return gson.toJson(propertyArray);
     }
 
+    protected String buildResponseBody(GalasaProperty property){
+        /*
+         * Builds a json array object from a single GalasaProperty containing a property
+         */
+        JsonArray propertyArray = new JsonArray();
+        if (property != null){
+            propertyArray.add(property.toJSON());
+        }
+        return gson.toJson(propertyArray);
+    }
```

Comment:
See https://github.com/galasa-dev/framework/pull/470/files#r1387789428 - we can plug a `List<GalasaProperty>` into `gson.toJson`